src/main/java/tornadofx/CSS.kt

```diff
@@ -1042,30 +1042,27 @@ class InlineCss : PropertyHolder(), Rendered {
 fun Iterable<Node>.style(append: Boolean = false, op: InlineCss.() -> Unit) = forEach { it.style(append, op) }
 
 fun Styleable.style(append: Boolean = false, op: InlineCss.() -> Unit) {
-    val block = InlineCss().apply(op)
-
-    fun setter(value: String) = when (this) {
-        is Node -> style = value
-        is MenuItem -> style = value
-        is PopupControl -> style = value
-        is Tab -> style = value
-        is TableColumnBase<*, *> -> this.style = value
-        else -> throw IllegalArgumentException("Don't know how to set style for Styleable subclass ${this@style.javaClass}")
-    }
+    val setStyleMethod = this.javaClass.methods.firstOrNull { method -> method.name == "setStyle" }
```

Comment:
We should probably get the method with correct name and correct parameter count/type.